lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -135,6 +135,21 @@ std::vector<std::string> GetStrings(const llvm::json::Object *obj,
   return strs;
 }
 
+static std::string GetDescriptionTrimmed(lldb::SBValue &value) {
```

Comment:
add a comment for this function mentioning that this returns an empty string if the description is not available. You can also change the return type to optional<std::string>, which might be cleaner.

Reply:
Done.